templates/events/add-new-event.html

```diff
@@ -805,6 +810,30 @@ <h5>Add Sub-Event </h5>
   overflow-y: visible;
 }
 
+  .mytooltip {
+  position: relative;
+  display: inline-block;
+  border-bottom: 1px dotted black;
+}
+
+.mytooltip .mytooltiptext {
+  visibility: hidden;
+  width: 1200px;
+  background-color: black;
+  color: #fff;
+  {#text-align: center;#}
+  border-radius: 6px;
+  padding: 10px 0;
```

Comment:
```suggestion
  padding: 10px;
```
I think this is looking better with padding on all sides.